core/modules/user/store/actions.ts

```diff
@@ -2,378 +2,301 @@ import { ActionTree } from 'vuex'
 import * as types from './mutation-types'
 import rootStore from '@vue-storefront/core/store'
 import i18n from '@vue-storefront/i18n'
-import { adjustMultistoreApiUrl } from '@vue-storefront/core/lib/multistore'
 import RootState from '@vue-storefront/core/types/RootState'
 import UserState from '../types/UserState'
 import { Logger } from '@vue-storefront/core/lib/logger'
-import { TaskQueue } from '@vue-storefront/core/lib/sync'
 import { UserProfile } from '../types/UserProfile'
-import { isServer, processURLAddress } from '@vue-storefront/core/helpers'
+import { isServer } from '@vue-storefront/core/helpers'
+import { UserService } from '@vue-storefront/core/data-resolver'
 import config from 'config'
 import EventBus from '@vue-storefront/core/compatibility/plugins/event-bus'
 import { StorageManager } from '@vue-storefront/core/store/lib/storage-manager'
 
 const actions: ActionTree<UserState, RootState> = {
-  async startSession (context) {
-    if (isServer || context.getters.isLocalDataLoaded) return
-    context.commit(types.USER_LOCAL_DATA_LOADED, true)
-
+  async startSession ({ commit, dispatch, getters }) {
     const user = localStorage.getItem(`shop/user/current-user`);
+    const usersCollection = StorageManager.get('usersCollection')
```

Comment:
After emrging https://github.com/DivanteLtd/vue-storefront/pull/3230 this will change to just  `user`